rust_src/src/remacs_sys.rs

```diff
@@ -154,6 +155,7 @@ extern "C" {
         dest: Lisp_Object,
         op: BoolVectorOp,
     ) -> Lisp_Object;
```

Comment:
You should not need to manually add functions. Instead, make sure the function is exported in a C header.